src/main/kotlin/no/nav/syfo/dialogmote/domain/Referat.kt

```diff
@@ -107,6 +107,7 @@ fun Referat.toJournalforingRequestArbeidstaker(
     dokumentName = this.toJournalpostTittel(moteTidspunkt),
     brevkodeType = BrevkodeType.DIALOGMOTE_REFERAT_AT,
     dokumentPdf = pdf,
+    varselUuid = UUID.randomUUID(),
```

Comment:
Burde vi egentlig bruke en referanse som er unik for dette spesifikke referatet til arbeidstaker (og tilvarende til behandler, arbeidsgiver) i stedet for å generere en random uuid? I teorien vil jo det samme referatet kunne journalføres flere ganger når man lager ny referanse hver gang.

Reply:
Da må vi vel i så fall gjøre større endringer? Alt er jo samme referat, så det er samme uuid. Alternativt må vi generere en uuid som blir lik for hver gang men unik for hver mottaker/referat 🤔

Reply:
Ja, lurer på om det går an å generere en uuid basert på en string - feks. referat.uuid og et eller annet som er unikt for mottaker som du sier.

Reply:
Java sin UUID godtar bare string som er på riktig format da, jeg finner ikke en som genererer uuid basert på en seed 🤷‍♂️

Reply:
Ny versjon nå, etter slack-samtale 👍